src/components/Button.tsx

```diff
@@ -1,31 +1,31 @@
-import React from "react";
-import { Text, TouchableOpacity } from "react-native";
+import React from 'react';
+import { Text, TouchableOpacity } from 'react-native';
+import handlePress from '../helper/onPressHelper';
+import { StyleSheet } from 'react-native';
 
 interface Props {
   title: string;
+  onPress: () => void;
 }
 
-const Button: React.FC<Props> = ({ title }) => {
-  const handlePress = () => {
-    console.log("Button pressed!");
-  };
-
+const Button: React.FC<Props> = ({ title, onPress }) => {
   return (
-    <TouchableOpacity onPress={handlePress} style={styles.button}>
+    <TouchableOpacity onPress={onPress} style={styles.button}>
       <Text style={styles.text}>{title}</Text>
     </TouchableOpacity>
   );
 };
 
-const styles = {
+const styles = StyleSheet.create({

```

Comment:
Can we find an alternative way to decouple styling from component logic? Maybe store stylesheets in a separate asset folder or something?

Reply:
I imagine that's definitely a possibility. I am okay with either way although this structure is the one chatgpt gave me lol

Reply:
i see. It's better to decouple to make the button component truly customizable.